.github/workflows/ci.yml

```
@@ -0,0 +1,29 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: ['3.0', '3.1', '3.2']
+        gemfile:
+          - gemfiles/rspec_3.9.gemfile
+          - gemfiles/rspec_3.10.gemfile
+          - gemfiles/rspec_3.11.gemfile
+          - gemfiles/rspec_3.12.gemfile
+          - gemfiles/rspec_3.13.gemfile
+    steps:
+    - uses: actions/checkout@v2
```

```suggestion
    - uses: actions/checkout@v4
```

i think this is the latest